test/expected/append.out

```diff
@@ -16,6 +16,14 @@ BEGIN
     RETURN '2017-08-22T10:00:00'::timestamptz;
 END;
 $BODY$;
+CREATE OR REPLACE FUNCTION now_s_parallel()
```

Comment:
This function is not marked parallel safe nor is it used anywhere.

Reply:
removed